README.md

```diff
@@ -20,12 +20,16 @@ You are expected to have Elixir and Postgres installed.
 
     mix deps.get
     mix ecto.setup
-    mix phoenix.server
+    npm install
```

Comment:
Oh, sorry, I guess "npm install" installs any npm deps. So this does make sense here.

I don't remember having to run this myself, but I guess it ran automatically when the app was generated.